apps/sms/views/conversation/js/compose.js

```diff
@@ -157,7 +157,6 @@ var Compose = (function() {
 
   function onSubjectVisibilityChanged() {
     if (subject.isVisible()) {
-      subject.focus();
       dom.form.classList.add('subject-composer-visible');
     } else {
       dom.message.focus();
```

Comment:
Still keeping this because we need to focus the message if the user removes the subject by pressing backspace. I'm not really worried that the focus management is not symetric for show and hide, because we don't focus the same element.